pkg/nsx/services/vpc/builder.go

```diff
@@ -115,6 +116,16 @@ func buildVpcAttachment(obj *v1alpha1.NetworkInfo, nsObj *v1.Namespace, cluster
 	return attachment, nil
 }
 
+// parse org id and project id from nsxProject path
+// example /orgs/default/projects/nsx_operator_e2e_test
+func nsxProjectPathToId(path string) (string, string, error) {
+	parts := strings.Split(path, "/")
+	if len(parts) < 4 {
+		return "", "", errors.New("invalid NSX project path")
+	}
+	return parts[2], parts[len(parts)-1], nil
+}
+
```

Comment:
We have this function in service common which is used by vpc service. Maybe we can remove this one to keep only one version of this function.